vue/components/ui/molecules/image-input/image-input.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+    <upload-area
+        class="image-input"
+        v-bind:files.sync="files"
+        v-bind:multiple="multiple"
+        v-bind:accept="'image/*'"
+        v-on:update:files="loadFiles"

```

Comment:
Can't this be a watcher over `files`? It's unusual to have both `sync` and `on:update` for the same var.